<launch>

  <node pkg="learned_map" type="mapCutter" name="mapCutter" output="screen">

  <param name="loadMapName" value="/home/yh/mapModel/01.11/test.vtk"/> 
  <param name="cutX" value="-20"/>
  <param name="cutY" value="-200"/>

  <param name="saveTrainName" value="/home/yh/mapModel/05.04/Train_Test/trainCloud.vtk"/>
  <param name="saveTestName" value="/home/yh/mapModel/05.04/Train_Test/testCloud.vtk"/>

  </node>
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find learned_map)/cfg/mapCutter.rviz" output="screen"/>
</launch>
